homeassistant/components/august.py

```diff
@@ -143,10 +143,15 @@ def setup(hass, config):
     conf = config[DOMAIN]
     try:
         api_http_session = Session()
+        API_SESSIONS.append(api_http_session)
```

Comment:
We shouldn't use globals

Reply:
@balloob 
Used switch/netio.py as an example when doing this. :-) Found another example (zeroconf) that does it without a global. :-)